OracleHTTPServer/README.md

```diff
@@ -1,6 +1,6 @@
 # Oracle HTTP Server in containers
 ===================================
-This project includes quick start dockerfiles and samples for standalone Oracle HTTP Server based on Oracle Linux and Oracle JDK 8 (Server).
+This project includes quick start dockerfiles and samples for standalone Oracle HTTP Server 12.2.1.4 and JDK 8 and 14.1.1.0 bnd JDK 17 and 21 based on Oracle Linux 8 and 9.
```

Comment:
Does this project include content for both 12.2.1.4.0 and 14.1.1.0? Is 14.1.1.0 a typo for 14.1.2.0.0?

Reply:
That is a typo. that I thought I had corrected

Reply:
Look at the file it is outdated, this has been corrected